packages/icons/src/Icon.js

```diff
@@ -12,7 +12,7 @@ const Icon = ({ name, title, desc, titleId, descId, ...props }) => {
 
   if (!Component) {
     if (process.env.NODE_ENV !== 'production') {
-      console.trace(`icon ${iconName} does not exist`)
+      console.trace(`icon${name ? ' ' + name + ' ' : ' '}does not exist`)
```

Comment:
Could be made a little simpler if you add a space before the template literal.

```console.trace(`icon ${name ? name + ' ' : ''}does not exist`)```

Reply:
Alternatively, you could do this as well: ```console.trace(name ? `icon ${name} does not exist` : 'icon is missing name prop')```

Reply:
Went with the second version. Liked the clarity of the message. Very useful. Went ahead and commited/pushed the changes, so the PR should be up-to-date with the changes.